ci/build_container/build_recipes/nghttp2.sh

```diff
@@ -2,12 +2,9 @@
 
 set -e
 
-# Use master branch, which contains a fix for the spurious limit of 100 concurrent streams:
-# https://github.com/nghttp2/nghttp2/commit/2ba1389993729fcb6ee5794ac512f2b67b29952e
-# TODO(PiotrSikora): switch back to releases once v1.33.0 is out.
-VERSION=e5b3f9addd49bca27e2f99c5c65a564eb5c0cf6d  # 2018-06-09
+VERSION=1.33.0
 
-curl https://github.com/nghttp2/nghttp2/archive/"$VERSION".tar.gz -sLo nghttp2-"$VERSION".tar.gz
+curl https://github.com/nghttp2/nghttp2/releases/download/v"$VERSION"/nghttp2-"$VERSION".tar.gz -sLo nghttp2-"$VERSION".tar.gz
```

Comment:
Could we add a SHA256 verification step here? Should be a quick one liner.

Reply:
We aren't sha256 verifying anything in `ci/build_container/build_recipes/*`. I think I know how to do this - will report back.

Reply:
@htuch the commit to add sha256 verification is at https://github.com/envoyproxy/envoy/pull/4353/commits/28f0f5867c380663ec6367061b6fd06d6e2b6818 but the change is failing for the Mac build as `sha256sum` doesn't exist. I'll do some testing with the solutions at https://github.com/ESGF/esg-search/issues/84